### DISCUSSION TITLE
WIFI Update help please

### DISCUSSION THREAD

**Discussion by Billbrad77** [mcguirepr89/BirdNET-Pi#1235] [Q&A]
How can I update the local wifi settings on my existing BirdNET pi (3).  It has been moved to a new router with different SSID and pwds.   I love it and miss it.  It is a headless unit.

With a local monitor and keyboard I ran "sudo iwlist wlan0 scan"  and it tells me 

wlan0 interface doesn't support scanning:  Device or resource busy

Do I need to stop the Birdnet processes first?

Hopefully I don't need to make a new SD card, but any help would be appreciated.

THanks,

Bill

**Comment by rdz-oss** [comment]
Hi Bill,

you might be able to use raspi-config. https://raspberrytips.com/raspberry-pi-wifi-setup/#using-the-raspiconfig-tool
You can set /add a new wireless network.
BirdNET-Pi can keep running it should not be affected.

BR
Richard
